module/VuFind/src/VuFind/Db/Row/User.php

```diff
@@ -296,8 +296,7 @@ public function checkEmailVerified()
      */
     public function getTags($resourceId = null, $listId = null, $source = null)
     {
-        return $this->getDbService(TagServiceInterface::class)
-            ->getUserTagsFromFavorites($this, $resourceId, $listId, $source);
+        return $this->getDbTable('Tags')->getListTagsForUser($this->getId(), $resourceId, $listId, $source);
```

Comment:
Have to roll these back to take advantage of the legacy null case sensitivity value.